docs/resources.html

```diff
@@ -37,59 +40,62 @@ <h2 id="general">General</h2>
+				<p>Research communities which adopt diversity as one of their goals should avoid over-burdening under-represented scholars with diversity-related tasks. Under-represented scholars are sometimes expected to play a disproportionate role in advancing diversity and inclusion in institutions (<a href="https://www.nature.com/articles/s41559-019-0911-5" target="_blank">Jimenez et al., 2019</a>, or read this <a href="https://www.insidehighered.com/news/2019/06/04/whos-doing-heavy-lifting-terms-diversity-and-inclusion-work" target="_blank" title="Jiminez et al. (2019) summary">summary</a>). This may also be the case in journal editorial teams, and hence it is likely worthwhile to keep a note of who is advancing diversity and inclusion in your team to ensure a fair distribution of effort.</p>
 				<h3 id="english-language-editing">English language editing</h3>
 				<p>If your journal publishes in English, you can consider offering an additional free editing service for manuscripts submitted by those for whom English is a second language. For example, see the <a href="https://onlinelibrary.wiley.com/page/journal/14756811/homepage/forauthors.html" target="_blank">International Section guidelines</a> at <a href="https://onlinelibrary.wiley.com/journal/14756811" target="_blank">Personal Relationships</a> and this article about <a href="https://view.publitas.com/iarr/rrn-printernationalscholarship-november2021/page/1" target="_blank">how the International Section is contributing to elevating international scholarship</a>.</p>
+				<h3 id="retroactive-name-changes">Retroactive name changes</h3>
+				<p>Although retroactive name changes may seem strangely placed in a section on diversity, the historical inability to retroactively change names in publications is likely to disproportionately impact marginalized individuals. Women are historically more likely than men to change their name after marriage and/or divorce. Although many authors who have changed their names after marriage or divorce have no problem authoring publications under different names, for others viewing their previous name will be traumatic (e.g., where a marriage ended because of domestic violence). In addition, transgender individuals often choose to change their names, and being referred to by their previous name can also be traumatic for them.</p>
```

Comment:
```suggestion
				<p>Although retroactive name changes may seem strangely placed in a section on diversity, the historical inability to retroactively change names in publications is likely to disproportionately impact marginalized individuals. Women are historically more likely than men to change their name after marriage and/or divorce. Although many authors who have changed their names after marriage or divorce have no problem authoring publications under different names, for others viewing their previous name will be traumatic (e.g., where a marriage ended because of domestic violence). In addition, transgender individuals often choose to change their names, and being referred to by their previous name ("deadnaming") can be traumatic for them.</p>
```